### PR TITLE
Move to yml

### PR DESCRIPTION
**Pull request by TylerLeonhardt**

### CODE REVIEW

File: tools/releaseBuild/setVstsVariables.ps1 [hunk 1]
```diff
@@ -1,5 +1,5 @@
 $vstsVariables = @{
-    PSES_BRANCH = 'master'
+    PSES_BRANCH = 'move-to-yml'
```
Comment (rjmholt) [inline tools/releaseBuild/setVstsVariables.ps1]: Will this need changing back at some point?

File: vscode-powershell.build.ps1 [hunk 2]
```diff
@@ -33,6 +33,7 @@ task ResolveEditorServicesPath -Before CleanEditorServices, BuildEditorServices,
     }
     else {
         $script:psesRepoPath = Resolve-Path $script:psesRepoPath
+        gci $env:BUILD_SOURCESDIRECTORY
```
Comment (rjmholt) [inline vscode-powershell.build.ps1]: Should this be here? If so, can it be `Get-ChildItem` instead?

Comment (TylerLeonhardt) [reply vscode-powershell.build.ps1]:
```suggestion
```

File: tools/releaseBuild/setVstsVariables.ps1 (hunk 1 above)
Comment (TylerLeonhardt) [reply tools/releaseBuild/setVstsVariables.ps1]:
```suggestion
    PSES_BRANCH = 'master'
```

**Pull request merged**